data/2018-10/codeinthedark.md

```diff
@@ -0,0 +1,40 @@
+---
+id: code-in-the-dark-1-bkk-2018
+date: '2018-10-10'
+time: 19:00 ~ 22:00
+categories:
+  - Codefest
+topics:
+  - Frontend
+  - HTML
+  - CSS
+location:
+  title: cøde CRAFT Bar & Studio
+  url: https://www.google.com/maps/place/c%C3%B8de+CRAFT+Bar+%26+Studio/@13.8065307,100.5223342,15z/data=!4m2!3m1!1s0x0:0xe9eec03b79e68253?sa=X&ved=2ahUKEwic4camq-XdAhUKvo8KHQrmB9sQ_BIwGHoECAoQCw
+links:
+  - type: website
+    url: https://www.facebook.com/StupidHackTH/
```

Comment:
This link is not needed — this event is not really affiliated with StupidHackTH, I am just reusing its Event Pop organizer account ;)

Reply:
ok

Reply:
Did you forget to remove this link item from the data file?